scripts/yrd/install

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+# get previous location - should be 
+START_DIR="$(pwd)"
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+mkdir "$BASEDIR/tmp"
+cd "$BASEDIR/tmp"
+git clone https://github.com/kpcyrd/yrd.git
+cd yrd
+# install dependencies
```

Comment:
For consistency with other files in this repo, comments are formatted like this:

`# Install dependencies`

With `#` then space then capital unless it's a name that starts with a lowercase letter (such as yrd).